assets/js/googlesitekit/components-gm2/Button.js

```diff
@@ -120,7 +121,14 @@ const Button = forwardRef(
 			( icon && tooltipTitle && children === undefined )
 		) {
 			return (
-				<Tooltip title={ tooltipTitle }>{ ButtonComponent }</Tooltip>
+				<Tooltip
+					title={ tooltipTitle }
+					{ ...( tooltipEnterDelayInMS !== undefined
+						? { enterDelay: tooltipEnterDelayInMS }
+						: {} ) }
```

Comment:
While this works and is absolutely valid, it doesn't look like we use this conditional format anywhere else. Also, in my opinion, it is slightly less readable.

What do you think about setting a default value of the `tooltipEnterDelayInMS` prop to `100` (that seems to be [the default value in the API](https://mui.com/material-ui/api/tooltip/#Tooltip-prop-enterDelay)) and passing `enterDelay` unconditionally to `<Tooltip>` component?

```suggestion
					enterDelay: tooltipEnterDelayInMS
```

Reply:
@nfmohit Thanks, that makes sense